mutiny-core/src/node.rs

```diff
@@ -244,17 +365,11 @@ impl<S: MutinyStorage> Node<S> {
             .replace(chain_monitor.clone());
 
         // read channelmonitor state from disk
-        let channel_monitors = if empty_state {
```

Comment:
FYI: we've never set `empty_state` to `true` so I went ahead and deleted these references.

Reply:
oh yeah i think that was from the SCB stuff